Trappist/src/Promact.Trappist.DomainModel/ApplicationClasses/SingleMultipleQuestion.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Promact.Trappist.DomainModel.Models.Question;
+using System.Collections.Generic;
+
+namespace Promact.Trappist.DomainModel.ApplicationClasses
+{
+    public class SingleMultipleQuestion
+    {
+        public SingleMultipleAnswerQuestion singleMultipleAnswerQuestion { get; set; }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename property 'singleMultipleAnswerQuestion' to match camel case naming rules, consider using 'SingleMultipleAnswerQuestion'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS100)